keps/sig-auth/0014-dynamic-audit-configuration.md

```diff
@@ -192,11 +236,12 @@ called `withDynamicAudit`. Another conditional clause will be added where the ha
 [provisioned](https://github.com/kubernetes/apiserver/blob/master/pkg/server/config.go#L536) allowing for the proper feature gating.
 
 #### Policy Enforcement
```

Comment:
Do you want to add a note somewhere in here that this approach lets us prototype the flexible policy design as a CRD with a proxy backend?